PdfiumAndroid/src/main/cpp/include/fpdf_edit.h

```diff
@@ -126,6 +126,38 @@ FPDF_EXPORT FPDF_PAGE FPDF_CALLCONV FPDFPage_New(FPDF_DOCUMENT document,
 FPDF_EXPORT void FPDF_CALLCONV FPDFPage_Delete(FPDF_DOCUMENT document,
                                                int page_index);
 
+// Experimental API.
```

Comment:
Are these just due to the new version of pdfium?

Reply:
Yes